### PR TITLE
stability: fix some checks in ready fns

### PR DESCRIPTION
**Pull request by zyguan**
This PR fixes some issues in member ready check fns, @weekface @xiaojingchen , PTAL

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -424,14 +424,14 @@ func (oa *operatorActions) pdMembersReadyFn(tc *v1alpha1.TidbCluster) (bool, err
 			ns, pdSetName, *pdSet.Spec.Replicas, ns, tcName, replicas)
 		return false, nil
 	}
-	if pdSet.Status.ReadyReplicas != tc.Spec.PD.Replicas {
+	if pdSet.Status.ReadyReplicas != replicas {
```
Comment (xiaojingchen) [inline tests/actions.go]: why? 
the `pdSet.Status.ReadyReplicas` is not contained failure pods, so we can not use `replicas` to check

Comment (zyguan) [reply tests/actions.go]: I'm not sure,

File: tests/actions.go [hunk 2]
```diff
@@ -424,14 +424,14 @@ func (oa *operatorActions) pdMembersReadyFn(tc *v1alpha1.TidbCluster) (bool, err
...
 		glog.Infof("statefulset: %s/%s .status.ReadyReplicas(%d) != %d",
-			ns, pdSetName, pdSet.Status.ReadyReplicas, tc.Spec.PD.Replicas)
+			ns, pdSetName, pdSet.Status.ReadyReplicas, replicas)
 		return false, nil
 	}
-	if len(tc.Status.PD.Members) != int(tc.Spec.PD.Replicas) {
+	if len(tc.Status.PD.Members) != int(replicas) {
```
Comment (xiaojingchen) [inline tests/actions.go]: ditto

File: tests/actions.go [hunk 3]
```diff
@@ -548,11 +558,16 @@ func (oa *operatorActions) tidbMembersReadyFn(tc *v1alpha1.TidbCluster) (bool, e
 			ns, tidbSetName, *tidbSet.Spec.Replicas, replicas)
 		return false, nil
 	}
-	if tidbSet.Status.ReadyReplicas != tc.Spec.TiDB.Replicas {
+	if tidbSet.Status.ReadyReplicas != replicas {
```
Comment (xiaojingchen) [inline tests/actions.go]: ditto

File: tests/actions.go (hunk 2 above)
Comment (zyguan) [reply tests/actions.go]: I see, I will revert theses changes.

**Comment by zyguan** [comment]
Most of fixes seems to be invalid. Sorry for bothering.

**Comment by gregwebs** [comment]
Maybe we can add some comments in the code to make it clearer.

**Pull request closed without merging**